### ISSUE TITLE
Missing identifiers in global lists not detected

### ISSUE THREAD

**Issue by Sei-Lisa** [Makopo/lslint#53]
Example:

```lsl
list A = [B];
default{timer(){A;}}
```

reports no errors. On the other hand:

```lsl
integer B = 1;
list A = [B];
default{timer(){A;}}
```

reports B as unused identifier in line 1.

<s>This looks like a consequence of c399e93 and its follow-up commits.</s> I'm looking into it.

Edit: Upon further analysis, it's clear that it's not a consequence of it. The first commit has the same problem.
